### PR TITLE
Create CSS Background Guide

### PR DESCRIPTION
**Pull request by D-A-Ledford**
Add Guide on CSS background colors and images.

### CODE REVIEW

File: src/pages/css/background/index.md [hunk 1]
```diff
@@ -2,14 +2,131 @@
...
+The image is repeated by default so that it covers the entire element.
+```CSS
+body {
+  background-image: url (“barn.jpg”);
+}
+```
+![image](https://user-images.githubusercontent.com/26467304/31036366-eb1fc260-a539-11e7-835d-e3f935a22c86.png)
+
+### Background Image - The Repeat Property
+The back-ground image is repeated both vertically (up and down) and horizontally (across the web page) by default.
```
Comment (texas2010) [inline src/pages/css/background/index.md]:
```diff
- The back-ground image
+ The background image
```

File: src/pages/css/background/index.md [hunk 2]
```diff
@@ -2,14 +2,131 @@
...
+  background-image: url (“barn.jpg”);
+  background-repeat: repeat-x;
+}
+```
+
+You can also use the background repeat property to set an image to not repeat.
+```CSS
+body {
+  background-image: url (“barn.jpg”);
+  background - repeat: no-repeat;
```
Comment (texas2010) [inline src/pages/css/background/index.md]:
```diff
- background - repeat: no-repeat;
+ background-repeat: no-repeat;
```

File: src/pages/css/background/index.md [hunk 3]
```diff
@@ -2,14 +2,131 @@
...
+* A valid color name such as “blue”
+* A HEX value such as “#FFFFF” (This is the hex value for white.)
+* A RGB value such as “rgb(76,175,80)” (This is the RGB value for light green.)
+
+### Background Images
+You can use the background image property to set an image as a background for an element. 
+The image is repeated by default so that it covers the entire element.
+```CSS
+body {
+  background-image: url (“barn.jpg”);
```
Comment (texas2010) [inline src/pages/css/background/index.md]:
```diff
- background-image: url (“barn.jpg”);
+ background-image: url(“barn.jpg”);
```

File: src/pages/css/background/index.md [hunk 4]
```diff
@@ -2,14 +2,131 @@
...
+![image](https://user-images.githubusercontent.com/26467304/31036366-eb1fc260-a539-11e7-835d-e3f935a22c86.png)
+
+### Background Image - The Repeat Property
+The back-ground image is repeated both vertically (up and down) and horizontally (across the web page) by default.
+You can use the background- repeat property to repeat the image vertically or horizontally.
+
+Here is an example that repeats the image vertically.
+```CSS
+body {
+  background-image: url (“barn.jpg”);
```
Comment (texas2010) [inline src/pages/css/background/index.md]:
```diff
- background-image: url (“barn.jpg”);
+ background-image: url(“barn.jpg”);
```

File: src/pages/css/background/index.md [hunk 5]
```diff
@@ -2,14 +2,131 @@
...
+  background-repeat: repeat-y;
+}
+```
+![vertical](https://user-images.githubusercontent.com/26467304/31039770-8962c7a6-a54e-11e7-9d25-4fb09760d219.PNG)
+
+You can repeat the image horizontally by setting the background repeat property to “repeat-x”.
+```CSS
+body {
+  background-image: url (“barn.jpg”);
```
Comment (texas2010) [inline src/pages/css/background/index.md]:
```diff
- background-image: url (“barn.jpg”);
+ background-image: url(“barn.jpg”);
```

File: src/pages/css/background/index.md [hunk 6]
```diff
@@ -2,14 +2,131 @@
...
+  background-repeat: repeat-x;
+}
+```
+
+You can also use the background repeat property to set an image to not repeat.
+```CSS
+body {
+  background-image: url (“barn.jpg”);
```
Comment (texas2010) [inline src/pages/css/background/index.md]:
```diff
- background-image: url (“barn.jpg”);
+ background-image: url(“barn.jpg”);
```

File: src/pages/css/background/index.md [hunk 7]
```diff
@@ -2,14 +2,131 @@
...
+  background - repeat: no-repeat;
+}
+```
+![norepeat](https://user-images.githubusercontent.com/26467304/31039801-c8761efc-a54e-11e7-8bb9-ec5b88885a50.PNG)
+
+### Background Image – The Position Property
+You can use the position property to specify where your image will be located on a web page.
+```CSS
+body {
+  background-image: url (“barn.jpg”);
```
Comment (texas2010) [inline src/pages/css/background/index.md]:
```diff
- background-image: url (“barn.jpg”);
+ background-image: url(“barn.jpg”);
```

File: src/pages/css/background/index.md [hunk 8]
```diff
@@ -2,14 +2,131 @@
...
+}
+```
+![position](https://user-images.githubusercontent.com/26467304/31039828-077d1038-a54f-11e7-8aa6-092253ca92b8.PNG)
+
+### Background Image – The Fixed Position
+You can use the background attachment property to set an image to a fixed position.
+A fixed position makes it so an image does not scroll with the rest of the page.
+```CSS
+body {
+  background-image: url (“barn.jpg”);
```
Comment (texas2010) [inline src/pages/css/background/index.md]:
```diff
- background-image: url (“barn.jpg”);
+ background-image: url(“barn.jpg”);
```

File: src/pages/css/background/index.md [hunk 9]
```diff
@@ -2,14 +2,131 @@
...
+  background - repeat: no-repeat;
```
Comment (texas2010) [inline src/pages/css/background/index.md]:
```diff
- background - repeat: no-repeat;
+ background-repeat: no-repeat;
```

**Comment by D-A-Ledford** [comment]
Thanks, I have fixed all the issues you listed. Let me know if you see any other issues.

**Comment by texas2010** [comment]
thanks. good job!

**Pull request merged**